client/client.go

```diff
@@ -105,6 +106,22 @@ type Lister interface {
 	List() ([]string, error)
 }
 
+type Subscription struct {
+	Source string `json:"source" yaml:"source"`
+	Type string `json:"type" yaml:"type"`
+	Broker string `json:"broker" yaml:"broker"`
```

Comment:
Maybe we don't want to display broker now, probably we could assume `default` for now, WDYT?

Reply:
That sounds like a good idea to me.  I expect we'll want to expose the concept later, but for now, simpler the better.  Once we expose something, we're stuck with it.  Better to start small and expose as-needed.